Oxide.Core/RemoteConsole/RemoteConsole.cs

```diff
@@ -106,6 +114,39 @@ private void OnMessage(MessageEventArgs e)
             }
         }
 
+        private class RconPlayer
+        {
+            public string SteamId { get; private set; }
+            public string Name { get; private set; }
+            public string Address { get; private set; }
+            public float Health { get; private set; }
+            public float MaxHealth { get; private set; }
+            public int Ping { get; private set; }
+            public string Lang { get; private set; }
+
+            public RconPlayer(IPlayer player)
```

Comment:
As far as I know it's alright to use Covalence here, but keep in mind that it loads after Oxide.Core so it may not be available right away and may cause an errors if used too early.